health-sdk/example-app/src/main/java/net/gini/android/health/sdk/exampleapp/invoices/ui/InvoicesActivity.kt

```diff
@@ -164,13 +233,17 @@ class InvoicesAdapter(
 
     override fun onBindViewHolder(viewHolder: ViewHolder, position: Int) {
         val invoiceItem = dataSet[position]
+
         viewHolder.recipient.text = invoiceItem.recipient ?: ""
         viewHolder.dueDate.text = invoiceItem.dueDate ?: ""
         viewHolder.amount.text = invoiceItem.amount ?: ""
 
         viewHolder.paymentComponent.prepareForReuse()
         viewHolder.paymentComponent.isPayable = invoiceItem.isPayable
+        viewHolder.paymentComponent.identifier = invoiceItem.documentId
     }
 
     override fun getItemCount() = dataSet.size
-}
\ No newline at end of file
+}
+
+private const val REVIEW_FRAGMENT_TAG = "payment_review_fragment"
```

Comment:
I think this tag belongs to ReviewFragment. we can add it to the companion object of the ReviewFragment and use it from every place(we are declaring the same variable in ReviewActivity

Reply:
Great idea! The thing is that ReviewFragment is in the SDK and it's up to clients to decide what tag to use or if they use a tag at all. 

In the example app I added this tag constant twice because they belong to different activities.